views/default/admin/users/header.php

```diff
@@ -33,6 +33,8 @@
 	'action' => elgg_get_current_url(),
 	'disable_security' => true,
 	'class' => $search_class,
+	'role' => 'search',
+	'aria-label' => elgg_echo('aria:label:admin:user:search'),
```

Comment:
should it not be `admin:users:search`? extra s

Reply:
fixed